manager/controllers/default/header.php

```diff
@@ -77,8 +77,17 @@ public function render()
      */
     public function setPlaceholders()
     {
+
+        if($this->modx->getOption('manager_use_fullname') == true) {
+            $userProfile = $this->modx->user->getOne('Profile');
+            $username = $userProfile->get('fullname');
+        } else {
+            $username = '';
```

Comment:
Am i right to assume that if `manager_use_fullname` is set to `false` neither the username nor the full name would be available ?